src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -18,9 +13,14 @@
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
-using Microsoft.DotNet.Services.Utility;
 using Maestro.Contracts;
-using System.Collections.Immutable;
+using Microsoft.DotNet.Services.Utility;
```

Comment:
Why did these namespaces all rearrange? It doesn't seem to be in service of consistency (since the file above it is the original way)... and it's making this diff exceedingly large.

Reply:
I think my Visual Studio just followed whatever `.editorconfig` where the default is usually "BCL first, then the rest alphabetically". I guess some refactoring that was moving code around did that when re-generating the list of usings.